fedimint-bin-tests/src/main.rs

```diff
@@ -516,6 +517,51 @@ async fn latency_tests(dev_fed: DevFed) -> Result<()> {
     Ok(())
 }
 
+async fn reconnect_test(dev_fed: DevFed) -> Result<()> {
+    #[allow(unused_variables)]
+    let DevFed {
+        bitcoind,
+        cln,
+        lnd,
+        mut fed,
+        gw_cln,
+        gw_lnd,
+        electrs,
+        esplora,
+        process_mgr,
+    } = dev_fed;
+
+    bitcoind.mine_blocks(110).await?;
+    fed.await_block_sync().await?;
+    fed.await_all_peers().await?;
+
+    // test a peer missing out on epochs and needing to rejoin
+    fed.kill_server(0).await?;
+    // FIXME increase this number once we can avoid processing epoch messages too
+    // far in the future
+    fed.generate_epochs(1).await?;
+
+    let mut fed = Federation::new(&process_mgr, bitcoind.clone(), 0..4).await?;
```

Comment:
does this work? 

it shouldn't because fedimintd-1,2,3 are still running

Reply:
I was confused as well as It's done the same way in `reconnect-test.sh`, it starts all the servers again. Should I change it to starting individual servers? (It does work though)

Reply:
also important: need to be careful about log files here. 
ProcessManager::spawn creates a log file. maybe it should open file in append mode.

Reply:
> Should I change it to starting individual servers?

yes that should be fine.